pkg/operator/ceph/cluster/crash/crash.go

```diff
@@ -186,3 +198,39 @@ func clusterOwnerRef(clusterName, clusterID string) metav1.OwnerReference {
 		BlockOwnerDeletion: &blockOwner,
 	}
 }
+
+func generateCrashEnvVar(v cephver.CephVersion) corev1.EnvVar {
+	val := fmt.Sprintf("-m $(ROOK_CEPH_MON_HOST) -k %s -n client.admin", keyring.VolumeMount().AdminKeyringFilePath())
+	if v.IsAtLeast(cephver.CephVersion{Major: 14, Minor: 2, Extra: 5}) {
+		val = fmt.Sprintf("-m $(ROOK_CEPH_MON_HOST) -k %s", keyring.VolumeMount().CrashCollectorKeyringFilePath())
```

Comment:
How about if the keyring gets mounted to the same place in the pod whether or not it's the admin or crash keyring? Then the only difference in this env var would be the `-n client.admin`, right?

Reply:
I was just following the established convention, even though I'm not a big fan of it. Will see what I can do here.

Reply:
The path will never the same anyways since keys have different names.